profiles/search.py

```diff
@@ -0,0 +1,44 @@
+import urllib.request
+import os
+import json
+
+class Search:
+    
+    def __init__(self):
+        self.currentEmployeePage = 1
+        self.currentUserPage = 1
```

Comment:
On second thought, this class should not contain state, because it could be called by any function. So remove the initiationzation function. Instead the currentEmployeePage and currentUserPage should be in the function call of get_linkedin_urls_google_search and get_user_profile_suggestions.